src/main/java/com/easypost/model/ErrorDeserializer.java

```diff
@@ -12,8 +12,32 @@
 
 import java.lang.reflect.Type;
 import java.util.ArrayList;
+import java.util.Map.Entry;
 
 public final class ErrorDeserializer implements JsonDeserializer<Error> {
+    /**
+     * Recursively traverse an error JSON element and its sub-element(s), and extracts all
+     * error string values found into the specified string list.
+     *
+     * @param element the JSON element to traverse
+     * @param messagesList the list of strings to append found values to
```

Comment:
question: this description doesn't make sense, the last 3 words seem to not belong here or are unfinished?

Reply:
`specified string list` here is the `messagesList`

Reply:
mmm re-reading this I just misunderstood it